src/sensorthings.js

```diff
@@ -22,9 +27,9 @@ module.exports = (config) => {
   let router = express.Router();
   router.use('/', baseRouter);
 
-  resourceEndpoints.forEach(endpoint => {
-    router.use(route.generate(endpoint),
-               require('./routes/' + Case.snake(endpoint)));
+  entities.forEach(endpoint => {
+    router.use(route.generate(endpoint), associations,
```

Comment:
Sorry, I can't see how would this work. `route.generate()` only generates the regexp for matching the endpoint url, it does not handle anything related to the router.